tools/reviewer/job/sync/GitHubClient.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Base64;
+
+import static java.net.HttpURLConnection.HTTP_CREATED;
+import static java.net.HttpURLConnection.HTTP_OK;
+
+public class GitHubClient {
+
+  private final ReviewerBot bot;
+
+  GitHubClient(ReviewerBot bot) {
+    this.bot = bot;
+  }
+
+  String postData(JSONObject json, String request) {
+    try {
+      URL url = new URL(request);
```

Comment:
I think you can reuse some code between `postData` and `getResponse`:
`getConnection(url) -> HttpURLConnection` that also adds the auth.
`readResponse(connection) -> String` that reads the response.
WDYT?

Reply:
Done.